cg/meta/upload/gt.py

```diff
@@ -126,3 +126,17 @@ def _is_variant_file(genotype_file: housekeeper_models.File):
 
     def _get_genotype_files(self, version_id: int) -> list:
         return self.hk.files(version=version_id, tags=["genotype"]).all()
+
+    @staticmethod
+    def genotype_check(case_obj: models.Family) -> bool:
+        """Check if balsamic case is contains WGS and normal sample"""
+
+        samples = [link.sample for link in case_obj.links]
```

Comment:
I think there should be a function for this in cg repeated list comprehension to keep this more DRY(do not repeat yourself):
Could you consider adding a new function in the Family model in: cg/store/models.py 

something like:
```
   @property
   def get_samples_in_case(self) -> List[models.Sample]:
        """Get samples in a case."""
        return [link.sample for link in self.links]
```

and then you modify this code also in cg/store/api/findbusinessdata.py to use your new function:
```
   def get_samples_by_family_id(self, family_id: str) -> List[models.Sample]:
        """Get samples on a given family_id"""
        case: models.Family = self.family(internal_id=family_id)
        return case.get_samples_in_case if case else []
```

and also import it in this module and use it directly:

```
samples:  List[models.Sample] = case_obj.get_samples_in_case
```

Reply:
I can definitely fix this!